_posts/2022-04-11-nwaku-update.md

```diff
@@ -0,0 +1,277 @@
+---
+layout: post
+name:  "What's new in `nwaku`?"
+title:  "What's new in `nwaku`?"
+date:   2022-04-11 10:00:00 +0200
+author: hanno
+published: true
+permalink: /nwaku-update-2022-04
+categories: research
+summary: A summary of recent developments in the Nim-based Waku v2 client and a preview of current and future focus areas.
+image: /assets/img/vac.png
+discuss:
+---
+
+## Background
+
+If you've been following our [research log](https://vac.dev/research-log/),
+you'll know that many things have happened in the world of Waku v2 since [our last general update](./2020-02-14-waku-update.md).
+In line with our [long term goals](https://vac.dev/#about),
+we've introduced new protocols,
+tweaked our existing protocols
+and expanded our team capacity.
+We've also shown [in a series of practical experiments](./2021-10-25-waku-v1-vs-waku-v2.md) that Waku v2 does indeed deliver on some of the [theoretical advantages](./2020-07-01-waku-v2-pitch.md) it was designed to have over its predecessor, Waku v1.
```

Comment:
These links are incorrect, need to use the one present in the `permalink` metadata of the target post.

Reply:
Thanks! Permalinks all fixed in [05b607c](https://github.com/vacp2p/vac.dev/pull/55/commits/05b607c2812a885741e7f51bc0b465aff978a26e)